tokens/quipuswap.whitelist.json

```diff
@@ -55,7 +57,8 @@
         "symbol": "wXTZ",
         "name": "Wrapped Tezos",
         "thumbnailUri": "https://raw.githubusercontent.com/StakerDAO/wrapped-xtz/dev/assets/wXTZ-token-FullColor.png"
-      }
+      },
+      "categories": ["Bridge"]
```

Comment:
It is not a bridge token. https://docs.stakerdao.com/wrapped-tezos-wxtz-faq-and-docs